Gemfile

```diff
@@ -27,7 +27,9 @@ gem "json-schema", "2.8.0"
 # Authentication
 
 gem "devise", "4.4.1"
+gem "devise-two-factor"
 gem "devise_lastseenable", "0.0.6"
+gem "rqrcode"
```

Comment:
Please add specific versions for the dependencies you add like everywhere else in the Gemfile, to keep it consistent :)